README.md

```diff
@@ -833,14 +833,18 @@ This will handle request `GET /customers/reqHeaders` as `select c from Customers
 Json request body support
 ---------------------
 
-Also, you can specify value for specification in json request body. It might be useful when you use large number of filters for request because request url is limited in size. Need to specify `paramType` with value `ParamType.BODY` (by default it set as `ParamType.QUERY`). For example:
+Also, you can specify value for specification in json request body. It might be useful when you use large number of filters for request because request url is limited in size. You can refer to the specification values by using `jsonPath` property of `@Spec` (instead of `params` property).
+
+<b>Warning!</b> RequestBody with specification values will be consumed during processing and will not be available for further operations!
```

Comment:
"... for further operations!" --> "... for further operations (i.e. ServletInputStream will return no data). If you need to use request body for something else (rather than just building the Specification), you need to use some kind of content caching/wrapping (e.g. in a servlet filter)"